docs/guides/serverless-hamsim-template.ipynb

```diff
@@ -0,0 +1,1087 @@
+   "id": "b1fde164-2b2b-473b-93b8-30168882076b",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# Build a function template for Hamiltonian simulation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4bbc25a8-5315-417f-bf35-7ac715ebdf98",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "version-info"
+    ]
+   },
+   "source": [
+    "<details>\n",
+    "<summary><b>Package versions</b></summary>\n",
+    "\n",
+    "The code on this page was developed using the following requirements.\n",
+    "We recommend using these versions or newer.\n",
+    "\n",
+    "```\n",
+    "qiskit[all]~=1.2.4\n",
+    "qiskit-ibm-runtime~=0.32.0\n",
+    "qiskit-serverless~=0.18.0\n",
+    "qiskit-ibm-catalog~=0.2.0\n",
+    "qiskit-addon-aqc-tensor[quimb-jax]~=0.1.0\n",
+    "mergedeep==1.3.4\n",
+    "```\n",
+    "</details>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5092090f-0fb8-436e-8099-3d3059367efc",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "By now you've seen a basic example of how to get started writing, uploading, and running a program with Qiskit Serverless. If you haven't, start with [What is Qiskit Serverless?](/guides/serverless) for some background information first.\n",
+    "\n",
+    "The workloads you are building for your own use cases are likely not as simple as the examples previously shown. For example, you might need to carefully consider what domain-level inputs and outputs are relevant for your application, how to make sure your program can be reusable across a range of those inputs, and what kind of information you need returned during the execution of the job so you can better evaluate the progress of your workload. You might even want to incorporate a \"dry run\" mode that lets you test the impact of a set of particular inputs before sending the resulting circuits to hardware.\n",
+    "\n",
+    "A function template is an example of a realistic workload that uses a specific application domain to contextualize these aspects. It is meant to be a starting point for you to modify for your own needs so you don't have to start from scratch.\n",
+    "\n",
+    "This guide walks through a function template for addressing Hamiltonian simulation problems."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "efde8ea3-8ba4-4859-b7cd-7c4e5c049783",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Write the function template\n",
+    "\n",
+    "First, write a function template for Hamiltonian simulation that uses the [AQC-Tensor Qiskit addon](https://qiskit.github.io/qiskit-addon-aqc-tensor/) to map the problem description to a reduced-depth circuit for execution on hardware.\n",
```

Comment:
The guide has been published, if it's better to link there: https://docs.quantum.ibm.com/guides/qiskit-addons-aqc